frontend/src/layout/scenes/components/SceneBreadcrumbs.tsx

```diff
@@ -11,27 +11,34 @@ import { Breadcrumb } from '~/types'
 interface SceneBreadcrumbBackButtonProps {
     forceBackTo?: Breadcrumb
     className?: string
+    iconOnly?: boolean
```

Comment:
GENERAL

I am unable to scroll this section using mouse

<img width="534" height="133" alt="image" src="https://github.com/user-attachments/assets/374aad78-5f8a-49ad-99b6-b93882e726cb" />